src/infinite-hits/infinite-hits.ts

```diff
@@ -32,7 +32,7 @@ import { noop } from '../utils';
       </div>
 
       <button
-        [class]="cx('showMore')"
+        [class]="showMoreClass"
```

Comment:
You can probably leverage [`NgClass`](https://angular.io/api/common/NgClass) directive to set the class name rather than relying on a getter.

Reply:
tried it but ngClass but couldn't get it work with computed properties. There is nowhere an example demonstrating it so I supposed 
```html
<div [ngClass]="{'my-class': true}" /> // works
<div [ngClass]="{[cx('my-class')]: true}" /> // produces a compile error
```
This workaround is also used in [menu widget]( https://github.com/algolia/angular-instantsearch/blob/develop/src/menu/menu.ts#L78-L86)

Reply:
Actually I can use `class="{{cx('loadMore')}} {{ this.state.isLastPage ? cx('loadMore', 'disabled') : ''}}"` which is maybe better?

Reply:
Ah. That's weird. Ok then you can use an array to avoid the extra space when it's not disabled.

Reply:
https://github.com/angular/angular/issues/13855